apis/medusa/v1alpha1/medusa_types.go

```diff
@@ -149,4 +149,43 @@ type MedusaClusterTemplate struct {
 	// See https://docs.datastax.com/en/developer/python-driver/latest/security/ for more information on the required files.
 	// +optional
 	CertificatesSecretRef corev1.LocalObjectReference `json:"certificatesSecretRef,omitempty"`
+
+	// Define the readiness probe settings to use for the Medusa containers.
+	// +optional
+	ReadinessProbeSettings *ProbeSettings `json:"readinessProbeSettings,omitempty"`
```

Comment:
Can we drop `Settings` off the end of these names. This is a total nit, but I think that e.g. `ReadinessProbe` is sufficient.

Reply:
Since we're not defining the whole probe (I'm leaving out the Exec part on purpose because it shouldn't be overridden), I think it's fair to leave "settings" at the end.